### DISCUSSION TITLE
Processing of .tif files

### DISCUSSION THREAD

**Discussion by GyDomonkos** [karttapullautin/karttapullautin#95] [Ideas]
For some reason the LIDAR data provided in Germany (Hessen to be exact) and the Company I purchased the data from for my map in Hungary was in .tif format with the ground points in one file and the rest in another file with the two .tfw files with them for georeferencing.

Would it be possible, to add a feature to rusty-pullauta, that would make it possible to process data like that? I personally think it would be a valuable feature.

Best regards, Doma

**Comment by rphlo** [comment]
Hi,
I'd recommend you use an external conversion tool to convert your tif file(s) to LAS or XYZ, then use those with this program.
I don't have the resource to create support for all of the imaginable Lidar formats existing.

**Reply by GyDomonkos** [reply]
I attempted to convert the files to xyz using QGIS, but rusty-pullauta gave an error, due to incorrect files. This means, that the conversion didnt quite work. The [georust/geotiff](https://github.com/georust/geotiff) repository provides a Rust library for reading GeoTiff files. I understand that integrating support for GeoTiff files is complex and may not seem worthwhile, but it would undoubtedly be valued by those working with Lidar Data in Germany on orienteering maps.

**Reply by rphlo** [reply]
From my understanding the Geotiff file does not contains all the info that a las does. A las file contains for example point classification that a geotiff lacks. Hence it seems not possible to use geotiffs as las replacements.
